http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/BuildHelmChartValuesSpec.scala

```diff
@@ -295,7 +295,8 @@ class BuildHelmChartValuesSpec extends AnyFlatSpecLike with LeonardoTestSuite {
       userEmail = userEmail2,
       stagingBucket = GcsBucketName("test-staging-bucket"),
       envVariables,
-      None
+      None,
+      Some(GcsBucketName("fc-bucket"))
```

Comment:
would be nice to see one of these test for `None`

Reply:
good idea. There are 4 tests for fc-bucket. I will change one of them to none